apps/backend/src/db/migrations/20260711120000_memo_scope.sql

```
@@ -0,0 +1,30 @@
+ALTER TABLE memos
+ADD COLUMN IF NOT EXISTS scope_user_id TEXT;
+
+ALTER TABLE memos
+ADD CONSTRAINT memo_scope_owner CHECK (
+    (scope = 'user' AND scope_user_id IS NOT NULL) OR
+    (scope <> 'user' AND scope_user_id IS NULL)
+);
+
+-- Drives the "About you" explorer filter and the per-owner retrieval gate:
+-- both look up a user's own private-tier memos.
+CREATE INDEX idx_memos_scope_user ON memos (workspace_id, scope_user_id) WHERE scope = 'user';
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)